package.json

```diff
@@ -164,17 +168,18 @@
 		"sprintf-js": "1.1.1",
 		"style-loader": "1.0.0",
 		"stylelint-config-wordpress": "13.1.0",
-		"typescript": "3.5.3",
+		"typescript": "3.8.2",
 		"uuid": "3.3.2",
 		"webpack": "4.41.0",
 		"worker-farm": "1.7.0"
 	},
 	"scripts": {
 		"prebuild": "npm run check-engines",
-		"clean:packages": "rimraf ./packages/*/build ./packages/*/build-module ./packages/*/build-style ./packages/*/node_modules",
+		"clean:packages": "rimraf ./packages/*/build ./packages/*/build-* ./packages/*/node_modules",
 		"prebuild:packages": "npm run clean:packages && lerna run build",
 		"build:packages": "node ./bin/packages/build.js",
-		"build": "npm run build:packages && wp-scripts build",
+		"build:package-types": "node ./bin/packages/build-types.js",
+		"build": "npm run build:packages && npm run build:package-types && wp-scripts build",
```

Comment:
I'm unsure where the types build should go.
- Part of `build:packages`?
- Concurrent/before/after?

Reply:
No clear preference. Part of `build:packages` might make sense, so they're seen as part of the package build, rather than something extraneous (that might fall through the cracks?).

(But I think it's also fine as-is, otherwise.)

Reply:
At least based on the current [packages release workflow](https://github.com/WordPress/gutenberg/blob/master/packages/README.md#releasing-packages), if these are to be included in the Lerna publish, it should happen as part of `build:packages`:

https://github.com/WordPress/gutenberg/blob/8a94254b6adac030f5a6ffd8e718972e1f19f48e/package.json#L204-L206

Otherwise, those `publish:*` commands would need to be updated as well. But I think it makes sense to consider this as part of `build:packages`. The `wp-scripts` portion of the build is fine to consider as separate from the packages themselves.

Reply:
I've moved it into build:packages (first in `&&` chain).